### ISSUE TITLE
Inconsistent skull meta

### ISSUE THREAD

**Issue by SlimeDog** [danirod12/NTD-LuckyBlock#8]
Paper 1.20.2-252 (or anything after build 249)
ntdLuckyBlock 2.8.12

On server start-up (ie, enable ntdLuckyBlock)
```
[15:40:27] [Server thread/INFO]: [ntdLuckyBlock] Loading NMS for your platform (git-Paper-252 (MC: 1.20.2), v1_20_R2)...
[15:40:27] [Server thread/WARN]: Found inconsistent skull meta, this should normally not happen and is not a Bukkit / Spigot issue, but one from a plugin you are using.
Bukkit will attempt to fix it this time for you, but may not be able to do this every time.
If you see this message after typing a command from a plugin, please report this to the plugin developer, they should use the api instead of relying on reflection (and doing it the wrong way).
```
for each luckyblock in the world. No other information was provided, but see https://github.com/PaperMC/Paper/issues/9875 for some hints.

**Comment by danirod12** [comment]
Hello. Thank you for reporting. Please check this update ([free](https://www.spigotmc.org/resources/92026/update?update=521399) or [premium](https://www.spigotmc.org/resources/94872/update?update=521400))

**Comment by SlimeDog** [comment]
Thanks. Confirmed fix in ntdLB 2.8.13.
